frontend/src/components/EmployeeModal/EmployeeInfo.tsx

```diff
@@ -11,6 +11,16 @@ type EmployeeInfoProps = {
   phone?: string;
 };
 
+// The component accepts firstName, lastName, netId, and phone as optional props,
```

Comment:
Nice documentation! But I think that instead of using single line comments `//`, you should wrap this documentation block in a JSDoc comment using `/** */` since this is the standard way of writing documentation in Javascript.